src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -203,6 +204,7 @@ def __init__(
         feature_extractor: Union["SequenceFeatureExtractor", str] = None,
         tokenizer: Optional[PreTrainedTokenizer] = None,
         decoder: Optional[Union["BeamSearchDecoderCTC", str]] = None,
+        image_processor: Optional["BaseImageProcessor"] = None,
```

Comment:
Do you need to add it as an argument here? Couldn't you just set `self.image_processor = None` directly below?

Reply:
I could, but chose to keep the same signature at the other pipeline init's. Happy to do it either way tbh